2022-11-17 11:24:27,874 werkzeug  * Restarting with watchdog (windowsapi) INFO
2022-11-17 11:24:34,632 werkzeug  * Debugger is active! WARNING
2022-11-17 11:24:34,632 werkzeug  * Debugger PIN: 707-636-620 INFO
2022-11-17 11:24:34,741 werkzeug  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit) INFO
2022-11-17 11:24:58,269 root Welcome to Youtube Scrapping INFO
2022-11-17 11:24:58,295 werkzeug 127.0.0.1 - - [17/Nov/2022 11:24:58] "GET / HTTP/1.1" 200 - INFO
2022-11-17 11:24:58,416 werkzeug 127.0.0.1 - - [17/Nov/2022 11:24:58] "[33mGET /static/css/main.css HTTP/1.1[0m" 404 - INFO
2022-11-17 11:25:38,728 werkzeug 127.0.0.1 - - [17/Nov/2022 11:25:38] "POST /details HTTP/1.1" 200 - INFO
2022-11-17 11:25:41,098 werkzeug 127.0.0.1 - - [17/Nov/2022 11:25:41] "POST /details HTTP/1.1" 200 - INFO
2022-11-17 11:27:27,695 werkzeug 127.0.0.1 - - [17/Nov/2022 11:27:27] "GET /details/qJJEf6p8xH0 HTTP/1.1" 200 - INFO
2022-11-17 11:37:44,236 werkzeug  * Restarting with watchdog (windowsapi) INFO
2022-11-17 11:37:50,794 werkzeug  * Debugger is active! WARNING
2022-11-17 11:37:50,810 werkzeug  * Debugger PIN: 707-636-620 INFO
2022-11-17 11:37:50,954 werkzeug  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit) INFO
2022-11-17 12:23:12,376 werkzeug  * Restarting with watchdog (windowsapi) INFO
2022-11-17 21:12:23,799 werkzeug  * Restarting with watchdog (windowsapi) INFO
2022-11-17 21:12:30,685 werkzeug  * Debugger is active! WARNING
2022-11-17 21:12:30,690 werkzeug  * Debugger PIN: 707-636-620 INFO
2022-11-17 21:12:30,769 werkzeug  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit) INFO
2022-11-17 21:13:29,083 root Welcome to Youtube Scrapping INFO
2022-11-17 21:13:29,100 werkzeug 127.0.0.1 - - [17/Nov/2022 21:13:29] "GET / HTTP/1.1" 200 - INFO
2022-11-17 21:13:29,195 werkzeug 127.0.0.1 - - [17/Nov/2022 21:13:29] "[33mGET /static/css/main.css HTTP/1.1[0m" 404 - INFO
2022-11-17 21:13:29,457 werkzeug 127.0.0.1 - - [17/Nov/2022 21:13:29] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - INFO
2022-11-17 21:14:00,258 root 'href' ERROR
Traceback (most recent call last):
  File "C:\Users\tando\PycharmProjects\Youtube_Master\main.py", line 135, in index
    all_videos.append(bigbox[i]['href'])
  File "C:\Users\tando\anaconda3\lib\site-packages\bs4\element.py", line 1519, in __getitem__
    return self.attrs[key]
KeyError: 'href'
2022-11-17 21:14:00,278 root list index out of range ERROR
Traceback (most recent call last):
  File "C:\Users\tando\PycharmProjects\Youtube_Master\main.py", line 156, in index
    link.append("www.youtube.com" + all_videos[i])
IndexError: list index out of range
2022-11-17 21:14:00,280 werkzeug  * Detected change in 'C:\\Users\\tando\\anaconda3\\Lib\\site-packages\\bs4\\element.py', reloading INFO
2022-11-17 21:14:01,230 root list index out of range ERROR
Traceback (most recent call last):
  File "C:\Users\tando\PycharmProjects\Youtube_Master\main.py", line 168, in index
    all_vids_new = all_videos[i].replace("/watch?v=", "")
IndexError: list index out of range
2022-11-17 21:14:01,239 werkzeug 127.0.0.1 - - [17/Nov/2022 21:14:01] "POST /details HTTP/1.1" 200 - INFO
2022-11-17 21:14:02,006 werkzeug  * Restarting with watchdog (windowsapi) INFO
2022-11-17 21:14:08,490 werkzeug  * Debugger is active! WARNING
2022-11-17 21:14:08,493 werkzeug  * Debugger PIN: 707-636-620 INFO
2022-11-17 21:14:08,572 werkzeug  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit) INFO
2022-11-17 21:14:44,042 root list index out of range ERROR
Traceback (most recent call last):
  File "C:\Users\tando\PycharmProjects\Youtube_Master\main.py", line 286, in download
    yt = YouTube(link[Downlink-1])
IndexError: list index out of range
2022-11-17 21:14:44,044 root local variable 'folder' referenced before assignment ERROR
Traceback (most recent call last):
  File "C:\Users\tando\PycharmProjects\Youtube_Master\main.py", line 304, in download
    gfile = drive.CreateFile({'parents': [{'id': folder}], 'title': title})
UnboundLocalError: local variable 'folder' referenced before assignment
2022-11-17 21:14:44,057 werkzeug  * Detected change in 'C:\\Users\\tando\\anaconda3\\Lib\\site-packages\\flask\\app.py', reloading INFO
2022-11-17 21:14:44,058 werkzeug  * Detected change in 'C:\\Users\\tando\\anaconda3\\Lib\\site-packages\\flask\\_compat.py', reloading INFO
2022-11-17 21:14:44,063 werkzeug 127.0.0.1 - - [17/Nov/2022 21:14:44] "[35m[1mGET /download/1 HTTP/1.1[0m" 500 - INFO
2022-11-17 21:14:44,518 werkzeug 127.0.0.1 - - [17/Nov/2022 21:14:44] "GET /download/1?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 - INFO
2022-11-17 21:14:44,519 werkzeug 127.0.0.1 - - [17/Nov/2022 21:14:44] "GET /download/1?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 - INFO
2022-11-17 21:14:44,548 werkzeug 127.0.0.1 - - [17/Nov/2022 21:14:44] "GET /download/1?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 - INFO
2022-11-17 21:14:44,557 werkzeug 127.0.0.1 - - [17/Nov/2022 21:14:44] "GET /download/1?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 - INFO
2022-11-17 21:14:44,575 werkzeug 127.0.0.1 - - [17/Nov/2022 21:14:44] "GET /download/1?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 - INFO
2022-11-17 21:14:46,095 werkzeug  * Restarting with watchdog (windowsapi) INFO
2022-11-17 21:14:52,514 werkzeug  * Debugger is active! WARNING
2022-11-17 21:14:52,517 werkzeug  * Debugger PIN: 707-636-620 INFO
2022-11-17 21:14:52,585 werkzeug  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit) INFO
2022-11-17 21:16:14,604 werkzeug  * Detected change in 'C:\\Users\\tando\\PycharmProjects\\Youtube_Master\\main.py', reloading INFO
2022-11-17 21:16:14,605 werkzeug  * Detected change in 'C:\\Users\\tando\\PycharmProjects\\Youtube_Master\\main.py', reloading INFO
2022-11-17 21:16:14,605 werkzeug  * Detected change in 'C:\\Users\\tando\\PycharmProjects\\Youtube_Master\\main.py', reloading INFO
2022-11-17 21:16:14,605 werkzeug  * Detected change in 'C:\\Users\\tando\\PycharmProjects\\Youtube_Master\\main.py', reloading INFO
2022-11-17 21:16:16,704 werkzeug  * Restarting with watchdog (windowsapi) INFO
2022-11-17 21:16:24,568 werkzeug  * Debugger is active! WARNING
2022-11-17 21:16:24,571 werkzeug  * Debugger PIN: 707-636-620 INFO
2022-11-17 21:16:24,687 werkzeug  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit) INFO
2022-11-17 21:16:28,050 root list index out of range ERROR
Traceback (most recent call last):
  File "C:\Users\tando\PycharmProjects\Youtube_Master\main.py", line 286, in download
    yt = YouTube(link[Downlink-1])
IndexError: list index out of range
2022-11-17 21:16:28,065 werkzeug 127.0.0.1 - - [17/Nov/2022 21:16:28] "[35m[1mGET /download/1 HTTP/1.1[0m" 500 - INFO
2022-11-17 21:16:28,626 werkzeug 127.0.0.1 - - [17/Nov/2022 21:16:28] "GET /download/1?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 - INFO
2022-11-17 21:16:28,627 werkzeug 127.0.0.1 - - [17/Nov/2022 21:16:28] "GET /download/1?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 - INFO
2022-11-17 21:16:28,641 werkzeug 127.0.0.1 - - [17/Nov/2022 21:16:28] "GET /download/1?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 - INFO
2022-11-17 21:16:28,647 werkzeug 127.0.0.1 - - [17/Nov/2022 21:16:28] "GET /download/1?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 - INFO
2022-11-17 21:16:28,687 werkzeug 127.0.0.1 - - [17/Nov/2022 21:16:28] "GET /download/1?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 - INFO
2022-11-17 21:16:45,010 werkzeug  * Restarting with watchdog (windowsapi) INFO
2022-11-17 21:16:51,480 werkzeug  * Debugger is active! WARNING
2022-11-17 21:16:51,483 werkzeug  * Debugger PIN: 707-636-620 INFO
2022-11-17 21:16:51,553 werkzeug  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit) INFO
2022-11-17 21:17:51,588 urllib3.connectionpool Connection pool is full, discarding connection: localhost. Connection pool size: 1 WARNING
2022-11-17 21:17:57,499 root 'href' ERROR
Traceback (most recent call last):
  File "C:\Users\tando\PycharmProjects\Youtube_Master\main.py", line 135, in index
    all_videos.append(bigbox[i]['href'])
  File "C:\Users\tando\anaconda3\lib\site-packages\bs4\element.py", line 1519, in __getitem__
    return self.attrs[key]
KeyError: 'href'
2022-11-17 21:17:57,544 root list index out of range ERROR
Traceback (most recent call last):
  File "C:\Users\tando\PycharmProjects\Youtube_Master\main.py", line 156, in index
    link.append("www.youtube.com" + all_videos[i])
IndexError: list index out of range
2022-11-17 21:17:58,485 root list index out of range ERROR
Traceback (most recent call last):
  File "C:\Users\tando\PycharmProjects\Youtube_Master\main.py", line 169, in index
    dict={"S.No":i+1,"Title":title[i].text,"Thumbnail": imga[i],"Link":all_vids_new}
IndexError: list index out of range
2022-11-17 21:17:58,499 werkzeug 127.0.0.1 - - [17/Nov/2022 21:17:58] "POST /details HTTP/1.1" 200 - INFO
2022-11-17 21:18:22,618 werkzeug 127.0.0.1 - - [17/Nov/2022 21:18:22] "POST /details HTTP/1.1" 200 - INFO
2022-11-17 21:18:32,057 root Downloading... INFO
2022-11-17 21:18:37,425 root Downloaded\U0001f60a\U0001f60a INFO
2022-11-17 21:18:53,095 oauth2client.client Successfully retrieved access token INFO
2022-11-17 21:18:53,104 googleapiclient.discovery_cache file_cache is only supported with oauth2client<4.0.0 INFO
2022-11-17 21:18:59,542 googleapiclient.http Encountered 403 Forbidden with reason "forbidden" WARNING
2022-11-17 21:18:59,543 root <HttpError 403 when requesting https://www.googleapis.com/upload/drive/v2/files?alt=json&uploadType=resumable returned "Insufficient permissions for the specified parent.". Details: "[{'domain': 'global', 'reason': 'forbidden', 'message': 'Insufficient permissions for the specified parent.', 'locationType': 'other', 'location': 'parent.permissions'}]"> ERROR
Traceback (most recent call last):
  File "C:\Users\tando\anaconda3\lib\site-packages\pydrive\files.py", line 368, in _FilesInsert
    metadata = self.auth.service.files().insert(**param).execute(
  File "C:\Users\tando\anaconda3\lib\site-packages\googleapiclient\_helpers.py", line 130, in positional_wrapper
    return wrapped(*args, **kwargs)
  File "C:\Users\tando\anaconda3\lib\site-packages\googleapiclient\http.py", line 902, in execute
    _, body = self.next_chunk(http=http, num_retries=num_retries)
  File "C:\Users\tando\anaconda3\lib\site-packages\googleapiclient\_helpers.py", line 130, in positional_wrapper
    return wrapped(*args, **kwargs)
  File "C:\Users\tando\anaconda3\lib\site-packages\googleapiclient\http.py", line 1093, in next_chunk
    return self._process_response(resp, content)
  File "C:\Users\tando\anaconda3\lib\site-packages\googleapiclient\http.py", line 1124, in _process_response
    raise HttpError(resp, content, uri=self.uri)
googleapiclient.errors.HttpError: <HttpError 403 when requesting https://www.googleapis.com/upload/drive/v2/files?alt=json&uploadType=resumable returned "Insufficient permissions for the specified parent.". Details: "[{'domain': 'global', 'reason': 'forbidden', 'message': 'Insufficient permissions for the specified parent.', 'locationType': 'other', 'location': 'parent.permissions'}]">

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\tando\PycharmProjects\Youtube_Master\main.py", line 302, in download
    gfile.Upload()
  File "C:\Users\tando\anaconda3\lib\site-packages\pydrive\files.py", line 285, in Upload
    self._FilesInsert(param=param)
  File "C:\Users\tando\anaconda3\lib\site-packages\pydrive\auth.py", line 75, in _decorated
    return decoratee(self, *args, **kwargs)
  File "C:\Users\tando\anaconda3\lib\site-packages\pydrive\files.py", line 371, in _FilesInsert
    raise ApiRequestError(error)
pydrive.files.ApiRequestError: <HttpError 403 when requesting https://www.googleapis.com/upload/drive/v2/files?alt=json&uploadType=resumable returned "Insufficient permissions for the specified parent.". Details: "[{'domain': 'global', 'reason': 'forbidden', 'message': 'Insufficient permissions for the specified parent.', 'locationType': 'other', 'location': 'parent.permissions'}]">
2022-11-17 21:18:59,548 werkzeug  * Detected change in 'C:\\Users\\tando\\anaconda3\\Lib\\site-packages\\pydrive\\files.py', reloading INFO
2022-11-17 21:18:59,556 werkzeug  * Detected change in 'C:\\Users\\tando\\anaconda3\\Lib\\site-packages\\pydrive\\auth.py', reloading INFO
2022-11-17 21:18:59,556 werkzeug  * Detected change in 'C:\\Users\\tando\\anaconda3\\Lib\\site-packages\\googleapiclient\\_helpers.py', reloading INFO
2022-11-17 21:18:59,557 werkzeug  * Detected change in 'C:\\Users\\tando\\anaconda3\\Lib\\site-packages\\googleapiclient\\http.py', reloading INFO
2022-11-17 21:18:59,585 werkzeug 127.0.0.1 - - [17/Nov/2022 21:18:59] "[35m[1mGET /download/8 HTTP/1.1[0m" 500 - INFO
2022-11-17 21:18:59,596 werkzeug 127.0.0.1 - - [17/Nov/2022 21:18:59] "GET /download/8?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 - INFO
2022-11-17 21:18:59,598 werkzeug 127.0.0.1 - - [17/Nov/2022 21:18:59] "GET /download/8?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 - INFO
2022-11-17 21:18:59,608 werkzeug 127.0.0.1 - - [17/Nov/2022 21:18:59] "GET /download/8?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 - INFO
2022-11-17 21:18:59,613 werkzeug 127.0.0.1 - - [17/Nov/2022 21:18:59] "GET /download/8?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 - INFO
2022-11-17 21:18:59,622 werkzeug 127.0.0.1 - - [17/Nov/2022 21:18:59] "GET /download/8?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 - INFO
2022-11-17 21:19:01,518 werkzeug  * Restarting with watchdog (windowsapi) INFO
2022-11-17 21:19:08,385 werkzeug  * Debugger is active! WARNING
2022-11-17 21:19:08,389 werkzeug  * Debugger PIN: 707-636-620 INFO
2022-11-17 21:19:08,496 werkzeug  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit) INFO
2022-11-17 21:19:57,562 werkzeug  * Restarting with watchdog (windowsapi) INFO
2022-11-17 21:20:03,732 werkzeug  * Debugger is active! WARNING
2022-11-17 21:20:03,737 werkzeug  * Debugger PIN: 707-636-620 INFO
2022-11-17 21:20:03,844 werkzeug  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit) INFO
2022-11-17 21:20:38,429 root 'src' ERROR
Traceback (most recent call last):
  File "C:\Users\tando\PycharmProjects\Youtube_Master\main.py", line 137, in index
    imga.append(img[i]['src'])
  File "C:\Users\tando\anaconda3\lib\site-packages\bs4\element.py", line 1519, in __getitem__
    return self.attrs[key]
KeyError: 'src'
2022-11-17 21:20:38,448 root list index out of range ERROR
Traceback (most recent call last):
  File "C:\Users\tando\PycharmProjects\Youtube_Master\main.py", line 156, in index
    link.append("www.youtube.com" + all_videos[i])
IndexError: list index out of range
2022-11-17 21:20:39,147 root list index out of range ERROR
Traceback (most recent call last):
  File "C:\Users\tando\PycharmProjects\Youtube_Master\main.py", line 169, in index
    dict={"S.No":i+1,"Title":title[i].text,"Thumbnail": imga[i],"Link":all_vids_new}
IndexError: list index out of range
2022-11-17 21:20:39,155 werkzeug 127.0.0.1 - - [17/Nov/2022 21:20:39] "POST /details HTTP/1.1" 200 - INFO
2022-11-17 21:20:46,996 root Downloading... INFO
2022-11-17 21:20:52,092 root Downloaded\U0001f60a\U0001f60a INFO
2022-11-17 21:21:01,428 oauth2client.client Successfully retrieved access token INFO
2022-11-17 21:21:01,432 googleapiclient.discovery_cache file_cache is only supported with oauth2client<4.0.0 INFO
2022-11-17 21:21:09,583 root Uploaded INFO
2022-11-17 21:21:10,786 root link shared INFO
2022-11-17 21:21:10,791 werkzeug 127.0.0.1 - - [17/Nov/2022 21:21:10] "GET /download/1 HTTP/1.1" 200 - INFO
2022-11-17 21:22:09,411 werkzeug  * Detected change in 'C:\\Users\\tando\\PycharmProjects\\Youtube_Master\\main.py', reloading INFO
2022-11-17 21:22:09,412 werkzeug  * Detected change in 'C:\\Users\\tando\\PycharmProjects\\Youtube_Master\\main.py', reloading INFO
2022-11-17 21:22:09,413 werkzeug  * Detected change in 'C:\\Users\\tando\\PycharmProjects\\Youtube_Master\\main.py', reloading INFO
2022-11-17 21:22:11,271 werkzeug  * Restarting with watchdog (windowsapi) INFO
2022-11-17 21:22:18,915 werkzeug  * Debugger is active! WARNING
2022-11-17 21:22:18,920 werkzeug  * Debugger PIN: 707-636-620 INFO
2022-11-17 21:22:19,031 werkzeug  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit) INFO
2022-11-17 21:22:35,640 werkzeug 127.0.0.1 - - [17/Nov/2022 21:22:35] "[35m[1mGET /details/tiolFUyXIEo HTTP/1.1[0m" 500 - INFO
2022-11-17 21:22:36,128 werkzeug 127.0.0.1 - - [17/Nov/2022 21:22:36] "GET /details/tiolFUyXIEo?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 - INFO
2022-11-17 21:22:36,129 werkzeug 127.0.0.1 - - [17/Nov/2022 21:22:36] "GET /details/tiolFUyXIEo?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 - INFO
2022-11-17 21:22:36,137 werkzeug 127.0.0.1 - - [17/Nov/2022 21:22:36] "GET /details/tiolFUyXIEo?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 - INFO
2022-11-17 21:22:36,149 werkzeug 127.0.0.1 - - [17/Nov/2022 21:22:36] "GET /details/tiolFUyXIEo?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 - INFO
2022-11-17 21:22:36,160 werkzeug 127.0.0.1 - - [17/Nov/2022 21:22:36] "GET /details/tiolFUyXIEo?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 - INFO
2022-11-17 21:23:47,061 werkzeug 127.0.0.1 - - [17/Nov/2022 21:23:47] "GET /details/tiolFUyXIEo HTTP/1.1" 200 - INFO
2022-11-17 21:26:45,114 werkzeug  * Detected change in 'C:\\Users\\tando\\anaconda3\\Scripts\\jupyter-notebook-script.py', reloading INFO
2022-11-17 21:26:47,009 werkzeug  * Restarting with watchdog (windowsapi) INFO
2022-11-17 21:26:53,742 werkzeug  * Debugger is active! WARNING
2022-11-17 21:26:53,745 werkzeug  * Debugger PIN: 707-636-620 INFO
2022-11-17 21:26:53,843 werkzeug  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit) INFO
